autolens/dataset/point_source.py

```diff
@@ -150,7 +147,7 @@ def from_dicts(cls, dicts: List[dict]) -> List[PointSourceDataset]:
         -------
         A collection of point source datasets.
         """
-        return cls(list(map(PointSourceDataset.from_dict, dicts)))
+        return cls(list(map(PointDataset.from_dict, dicts)))

```

Comment:
Constructor iterates first argument so no need to cast to list. Consider using typing.Iterable for the type annotation